.github/workflows/format.yml

```
@@ -35,6 +35,28 @@ jobs:
           node-version: 22
 
+      - name: 📦 Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.22
+
+      - name: 🦃 Install golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: latest
+          install-only: true
```

actionlint (1.7.7)

input "install-only" is not defined in action "golangci/golangci-lint-action@v6". available inputs are "args", "cache-invalidation-interval", "github-token", "install-mode", "only-new-issues", "problem-matchers", "skip-cache", "skip-save-cache", "version", "working-directory"

(action)